pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -76,9 +76,16 @@ const (
 	// MaxMonCount Maximum allowed mon count for a cluster
 	MaxMonCount = 9
 
+	// DefaultMsgr1PortName retains previous naming convention, but can be customized if needed
+    DefaultMsgr1PortName = "msgr1"
+
 	// DefaultMsgr1Port is the default port Ceph mons use to communicate amongst themselves prior
 	// to Ceph Nautilus.
 	DefaultMsgr1Port int32 = 6789
+
+	// DefaultMsgr1PortName retains previous naming convention, but can be customized if needed
+	DefaultMsgr2PortName = "msgr2"
```

Comment:
ditto, no need to export it.

Also, edit https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/mon/spec.go#L290 with this variable

Reply:
removed variables and hard coded istio compliant names

Reply:
Also, updated container port name as suggested.